src/uk/gov/defra/ffc/Tests.groovy

```diff
@@ -185,19 +185,18 @@ class Tests implements Serializable {
           ctx.withCredentials([
             ctx.usernamePassword(credentialsId: 'browserstack-credentials', usernameVariable: 'browserStackUsername', passwordVariable: 'browserStackAccessToken')
           ]) {
-            ctx.dir('./test/acceptance') {
-            ctx.sh('mkdir -p -m 777 html-reports')
-
-            def url = buildUrl(ctx, pr,  environment, repoName)
-            def envVars = []
-
+            def envVars = Provision.getPrQueueEnvVars(ctx, repoName, pr)
             envVars.push("BROWSERSTACK_USERNAME=${ctx.browserStackUsername}")
             envVars.push("BROWSERSTACK_ACCESS_KEY=${ctx.browserStackAccessToken}")
+            def url = buildUrl(ctx, pr, environment, repoName)
             envVars.push("TEST_ENVIRONMENT_ROOT_URL=https://${url}")
+            ctx.echo("$envVars")
+
+            ctx.dir('./test/acceptance') {
+            ctx.sh('mkdir -p -m 777 html-reports')
 
             ctx.withEnv(envVars) {
-            ctx.sh('docker-compose -f docker-compose.yaml build')
-            ctx.sh('docker-compose run wdio-cucumber')
+              ctx.sh('docker-compose up --build')
```

Comment:
This should still specify the `-f` with only `docker-compose.yaml` as the file chosen.  Otherwise this will automatically include `docker-compose.override.yaml` which may include local dev configuration that may not work in CI.  It may also bind ports that would prevent simultaneous builds.